tests/agents/test_default_height.py

```diff
@@ -0,0 +1,45 @@
+import holodeck
```

Comment:
These should just be in the same file for test_default_height, just put both functions in the same file. 
Also you should use the same config, just manually add the max height parameter in the test_max_height function. 
You can also share the same environment, which takes a little more work. I don't think it's necessary since its only two functions, but if you want you can look at Matt's pr.